universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -21,6 +21,7 @@
   /** Get the program block with the lowest index that has missing answer data if there is one. */
   Optional<Block> getFirstIncompleteBlock();
 
+  ImmutableList<SummaryData> getSummaryData();
```

Comment:
(blocking) this needs a docstring, and I think a space after to separate it from `preferredLanguageSupported`.